app/assets/javascripts/admin/addon/templates/web-hooks-edit.hbs

```diff
@@ -0,0 +1,105 @@
+      <div>
+        <RadioButton @class="subscription-choice" @name="subscription-choice" @value="individual" @selection={{this.model.webHookType}} />
+        {{i18n "admin.web_hooks.individual_event"}}
+        <InputTip @validation={{this.eventTypeValidation}} />
+      </div>
+
+      {{#unless this.model.wildcard_web_hook}}
+        <div class="event-selector">
+          {{#each this.eventTypes as |type|}}
+            <AdminWebHookEventChooser @type={{type}} @model={{this.model.web_hook_event_types}} />
+          {{/each}}
+        </div>
+      {{/unless}}
+
+      <div>
+        <RadioButton @class="subscription-choice" @name="subscription-choice" @value="wildcard" @selection={{this.model.webHookType}} />
+        {{i18n "admin.web_hooks.wildcard_event"}}
+      </div>
+    </div>
+
+    <div class="filters control-group">
+      <div class="filter">
+        <label>{{d-icon "circle" class="tracking"}}{{i18n "admin.web_hooks.categories_filter"}}</label>
+        <CategorySelector @categories={{this.model.categories}} @onChange={{action (mut this.model.categories)}} />
+        <div class="instructions">{{i18n "admin.web_hooks.categories_filter_instructions"}}</div>
+      </div>
+
+      {{#if this.showTagsFilter}}
+        <div class="filter">
+          <label>{{d-icon "circle" class="tracking"}}{{i18n "admin.web_hooks.tags_filter"}}</label>
+          <TagChooser @tags={{this.model.tag_names}} @everyTag={{true}} @excludeSynonyms={{true}} />
+          <div class="instructions">{{i18n "admin.web_hooks.tags_filter_instructions"}}</div>
+        </div>
+      {{/if}}
+
+      <div class="filter">
+        <label>{{d-icon "circle" class="tracking"}}{{i18n "admin.web_hooks.groups_filter"}}</label>
+        <GroupSelector @groupNames={{this.model.groupsFilterInName}} @groupFinder={{this.model.groupFinder}} />
+        <div class="instructions">{{i18n "admin.web_hooks.groups_filter_instructions"}}</div>
+      </div>
+    </div>
+
+    <PluginOutlet @name="web-hook-fields" @tagName="span" @connectorTagName="div" @args={{hash model=this.model}} />
+
+    <div>
+      <Input @type="checkbox" name="verify_certificate" @checked={{this.model.verify_certificate}} /> {{i18n "admin.web_hooks.verify_certificate"}}
+    </div>
+
+    <div>
+      <div>
+        <Input @type="checkbox" name="active" @checked={{this.model.active}} /> {{i18n "admin.web_hooks.active"}}
+      </div>
+
+      {{#if this.model.active}}
+        <div class="instructions">{{i18n "admin.web_hooks.active_notice"}}</div>
+      {{/if}}
+    </div>
+  </form>
+
+  <div class="controls">
+    <DButton
+      @translatedLabel={{this.saveButtonText}}
+      @action={{action "save"}}
+      @disabled={{this.saveButtonDisabled}}
+      @class="btn-default admin-webhooks__save-button"
+    />
+
+    {{#unless this.model.isNew}}
+      <DButton @class="btn-danger" @label="admin.web_hooks.destroy" @action={{action "destroy"}} />
```

Comment:
Can we move this Delete button and have it aligned to the right? It's quite prominent now and too close to the main actions (Edit/Show Events).